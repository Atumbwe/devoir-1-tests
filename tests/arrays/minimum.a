423 5 342 323 564 6 423 4 54 523423 4 5323 1 345 534 min