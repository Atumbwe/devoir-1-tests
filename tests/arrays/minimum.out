 ---------
 <  1  >
 ---------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
